boards/storm/Makefile

```diff
@@ -22,30 +26,51 @@ doc:
 .PHONY: target/sam4l/release/storm
 target/sam4l/release/storm:
 	@cargo build --release --target=sam4l.json
-	@$(OBJDUMP) $(OBJDUMP_FLAGS) $@ > target/sam4l/release/storm.lst
+	@$(SIZE) $@
+
+.PHONY: target/sam4l/debug/storm
+target/sam4l/debug/storm:
+	@cargo build --target=sam4l.json
+	@$(OBJDUMP) $(OBJDUMP_FLAGS) $@ > target/sam4l/debug/storm.lst
+	@$(SIZE) $@
 
 target/sam4l/release/storm.sdb: target/sam4l/release/storm
 	@tput bold ; echo "Packing SDB..." ; tput sgr0
 	@$(SLOAD) pack -m "$(SDB_MAINTAINER)" -v "$(SDB_VERSION)" -n "$(SDB_NAME)" -d $(SDB_DESCRIPTION) -o $@ $<
 
+target/sam4l/release/storm.lst: target/sam4l/release/storm
+	@$(OBJDUMP) $(OBJDUMP_FLAGS) $^ > $@
+
 target/sam4l/release/storm.hex: target/sam4l/release/storm
 	@$(OBJCOPY) -Oihex $^ $@
 
+target/sam4l/debug/storm.hex: target/sam4l/debug/storm
+	@$(OBJCOPY) -Oihex $^ $@
+
 .PHONY: clean
 clean:
 	@cargo clean
 
+.PHONY: debug
+debug: target/sam4l/debug/storm
```

Comment:
@alevy `make debug` fundamentally works for the storm platform, but fails in practice because without optimizations on, the size of the binary is too big for the ROM region as specified in the LD file. Right now the LD file allocates 0x20000 for it, and unoptimized the kernel is 0x28dfc in size (an optimized kernel is 0x13764 in size)

What are your thoughts about increasing the size?